CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Unreleased
 
 ### Added
+- Satoshi Font
+- Main navigation
+- Test:
+    - Main navigation test
```

Comment:
Here include also the ticket link, as we are doing here: https://github.com/Vizzuality/landgriffon/blob/dev/client/CHANGELOG.md